CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ unreleased
     showDefaultPaymentMethodFirst: false
   });
   ```
+- Apple Pay
+  - Fix issue where Apple Pay view was using `canMakePaymentsWithActiveCard` incorrectly
```

Comment:
Take it our leave it comment - is it worth noting that we're removing the use of `canMakePaymentsWithActiveCard` entirely?

Reply:
I think it's fine to keep it vague. Most merchants using Drop-in are using it so they don't actually have to think about how to set up things like Apple Pay. If you feel strongly about it, I can certainly change it.